lib/CustomFields/pages/ViewCustomFieldsRecord/ViewCustomFieldsRecord.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { connect } from 'react-redux';
+import { FormattedMessage } from 'react-intl';
+
+import { chunk } from 'lodash';
+
+import {
+  Accordion,
+  Callout,
+  KeyValue,
+  NoValue,
+  Row,
+  Col,
+} from '@folio/stripes-components';
+
+import {
+  selectModuleId,
+  selectOkapiData,
+} from '../../selectors';
+
+import {
+  useCustomFieldsFetch,
+  useSectionTitleFetch,
+  useLoadingErrorCallout,
+} from '../../utils';
+
+const propTypes = {
+  accordionId: PropTypes.string.isRequired,
+  backendModuleId: PropTypes.string,
+  backendModuleName: PropTypes.string.isRequired,
+  columnCount: PropTypes.number,
+  customFieldsValues: PropTypes.object.isRequired,
+  entityType: PropTypes.string.isRequired,
+  expanded: PropTypes.bool.isRequired,
+  okapi: PropTypes.shape({
+    tenant: PropTypes.string.isRequired,
+    token: PropTypes.string.isRequired,
+    url: PropTypes.string.isRequired,
+  }).isRequired,
+  onToggle: PropTypes.func.isRequired,
+};
+
+const defaultProps = {
+  columnCount: 4,
+};
+
+const ViewCustomFieldsRecord = ({
+  accordionId,
+  onToggle,
+  expanded,
+  okapi,
+  backendModuleId,
+  backendModuleName,
+  entityType,
+  customFieldsValues,
+  columnCount,
+}) => {
+  const {
+    customFields,
+    customFieldsLoaded,
+    customFieldsFetchFailed,
+  } = useCustomFieldsFetch(okapi, backendModuleId, entityType);
+  const {
+    sectionTitle,
+    sectionTitleLoaded,
+    sectionTitleFetchFailed,
+  } = useSectionTitleFetch(okapi, backendModuleName.toUpperCase());
+  const { calloutRef } = useLoadingErrorCallout(customFieldsFetchFailed || sectionTitleFetchFailed);
+  const accordionIsVisible = sectionTitleLoaded && customFieldsLoaded && customFields?.length > 0;
+
+  const formatedCustomFields = chunk(customFields, columnCount);
+  const defaultLabel = <FormattedMessage id="stripes-smart-components.customFields.recordAccordion.defaultName" />;
+
+  return (
+    <>
+      {accordionIsVisible
+        ? (
+          <Accordion
+            open={expanded}
+            id={accordionId}
+            onToggle={onToggle}
+            label={sectionTitle.value || defaultLabel}
+          >
+            {
+              formatedCustomFields.map((row, i) => (
+                <Row key={i}>
```

Comment:
Can we use ids or names from custom fields here as `key`?

Reply:
No, cause `formattedCustomFields` it's just `customFields` which splitted in groups by `chunk()` from lodash. This is done in order to correctly display columns